src/Components/Navbar/index.js

```diff
@@ -1,71 +1,63 @@
-  // Banana Portal: /home
-  // Profile Pic: /settings
-  // List: /all-clients (or /all-donors depending on what list is more popular)
+import Icon from '../Icon';
+import styles from './style.module.css';
 
-  const [dropdownOpen, setDropdownOpen] = useState(false);
+function BananaAdminNavbar() {
+  const [isOpen, setIsOpen] = useState(false);
 
-  const toggle = () => setDropdownOpen(!dropdownOpen);
-  // const [dropdownOpen, setDropdownOpen] = useState(false);
-  // const toggle = () => setDropdownOpen(!dropdownOpen);
+  const toggle = () => setIsOpen(!isOpen);
 
   return (
-    <BootstrapNavbar className={styles.container}>
-      <NavbarBrand href="/">
-        <Row className="align-items-center">
-          <Col>
-            <Icon name="bananaIcon" size={75} />
-          </Col>
-          <Col>
-            <NavbarText className={styles.bananaPortalNav}>
-              BANANA
-              <br />
-              PORTAL
-            </NavbarText>
-          </Col>
-        </Row>
+    // <div className={styles.container}>
+    <Navbar expand="sm" light className={styles.container}>
+      <NavbarBrand href="/" className={styles.navBarBrand}>
+        <Icon name="bananaIcon" size={75} className={styles.navIcon} />
+        <NavbarText className={styles.bananaPortalNav}>
+          BANANA
+          <br />
+          PORTAL
+        </NavbarText>
       </NavbarBrand>
-      <Nav pills>
-        <NavItem className={styles.navIcon}>
-          <NavLink href="/notifications">
-            <Icon name="alertBell" size={35} />
-          </NavLink>
-        </NavItem>
-        <NavItem className={styles.navIcon}>
-          <Dropdown nav isOpen={dropdownOpen} toggle={toggle}>
-            {dropdownOpen ? (
-              <DropdownToggle nav caret style={{ backgroundColor: '#F0EEEE' }}>
-                <Icon name="tasks" size={35} />
-              </DropdownToggle>
-            ) : (
-              <DropdownToggle nav caret>
-                <Icon name="tasks" size={35} />
-              </DropdownToggle>
-            )}
 
-            <DropdownMenu>
+      <NavbarToggler onClick={toggle} />
+
+      <Collapse isOpen={isOpen} navbar>
+        <Nav className={`ms-auto + ${styles.mobileNav}`} navbar>
+          <NavItem>
+            <NavLink href="/notifications">
+              <Icon name="alertBell" size={32} className={styles.navIcon} />
+              <NavbarText className={`${styles.displayMobile} + ${styles.navbarText}`}>
+                Notifications
+              </NavbarText>
+            </NavLink>
+          </NavItem>
+
+          <UncontrolledDropdown nav inNavbar className={styles.hideMobile}>
+            <DropdownToggle
+              nav
+              caret
+              className={`${styles.dropdownToggleMobile} + ${styles.navbarText}`}
+            >
+              <Icon name="tasks" size={32} className={styles.navIcon} />
+              <NavbarText className={styles.displayMobile}>Tasks</NavbarText>
```

Comment:
@tackc its kinda nit-picky but maybe these subnav links should span the full width of their parent container? if not, no worries ✔️ 
<img width="503" alt="image" src="https://github.com/FoodIsLifeBGP/banana-admin/assets/13320440/f496be6a-16c7-4cfe-80ba-9b24949c7452">

Reply:
Good catch! I also added the same hover styling to the `Notifications` and `Profile` links!